config/install/image.style.large.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 74996e0d-0fd0-4db9-8f72-7b0725ed7ce4
```

Comment:
We have an image styles module to share a number of common styles: https://github.com/SU-SWS/stanford_image_styles

I am open to dropping the module as it tends to couple things over the long term.

Reply:
these three came from core image module

Reply:
https://git.drupalcode.org/project/drupal/tree/8.8.x/core/modules/image/config/install